### DISCUSSION TITLE
Arm SIMD support

### DISCUSSION THREAD

**Discussion by llunved** [dalek-cryptography/curve25519-dalek#654] [Ideas]
The performance on Arm is pretty rough. It looks like the SIMD backend does not support Arm NEON, etc instructions. Has anyone looked at that, is there any broader interest? 

We are running Ampere, AWS Graviton 2 and 3 instances. On Graviton 2, signing take ~48 micros when I run bench....

**Comment by tarcieri** [comment]
See #457 though we've received [reports the NEON implementation is slower than the existing `u64` backend](https://github.com/dalek-cryptography/curve25519-dalek/pull/457#issuecomment-2015744264) on certain platforms.

**Reply by llunved** [reply]
Thank you. Apologies for missing that. Will try out and report back.
